meetings/2023-10-11.md

```diff
@@ -44,17 +57,62 @@
+  - PAC: This is a fair assessment, Solid heavily depends on HTTP, IETF has strong opinions on that and we should be discussing it with them.
+- make it clear that the WG will follow the W3C process and not the Solid process (2, FO)
+  - PAC: Solid process references a director which doesn't match latest W3C process. We should clarify that Solid WG will follow W3C process. And in fact the CG is already following in that regard.
+- less grandiose claim about what the WG will achieve (1, FO)
+  - PAC: We should clarify that we are not trying to solve the problem once and for all. This is a specific solution that various stakeholders are asking for.
+- editorial changes to charter (2)
+- WebID-based DID (1)
+
+* eP: Which of those CG can help with and which are W3C Team working on them?
+* PAC: Have not thought of that but fair question. Any opinion on those topics is welcome. We can discuss them here or reach out to me for additional information that can be improved on the charter. If you're asking, to what extent PRs on the charter be relevant. Feel free to ask but can't promise to accept them all. But it can be a start for discussion.
+* RG: I wanted to ask if those remarks are public or were submitted directly to the W3C team.
+* PAC: Members can choose to make the responses public but are not required. This is a synthesis of public and non-public responses.
+* SC: For some of the significant feedback requiring to update charter, clarify process, etc. 
+* SC: We need to update editors draft to be updated with current charter. If you see that some specific specs are not doing what they are supposed to do please let us know.
+* SC: Charter is referring to the TRs, to updated them we need to make a new release. WG can look at either editor drafts or latest publish versions on solidproject.org/TR
+* SC: I see most of the points as straight forward to address. Including broadening the scope. If you could come up with concrete issues we could propose PRs addressing those specific issues. We don't want to randomly propose things.
+* PAC: My goal is to create an issue for each one of those items. Whenever I can include more details I will do it. For public votes I will copy and paste all the details.
+* PAC: We probably want to use `solid/solid-wg-charter` for those PRs no to create to much activity on W3C charters repo.
+* PAC: I'm not worried about the contents of the cg.
+* eP: Regarding dependencies. Our latest approach to was to mark them as optional deliverables.
+* PAC: We don't have a clear list of deliverables. Depending on the maturity of the specs is transferring the authority to the CG. Here the criticism on different process between CG and WG is valid. We should think which specifications WG can bring to REC. We should decide for the rest if we make them soft or optional dependencies. 
+* eP: Extending the timeline, would it be feasible that the dependencies are necessary for Solid. Instead of depending on CG or whatever else.
```

Comment:
```suggestion
* PAC: We don't have a clear list of deliverables. Depending on the maturity of the specs, the authority may be transferred to the CG. Here the criticism on different process between CG and WG is valid. We should think which specifications WG can bring to REC. We should decide for the rest if we make them soft or optional dependencies. 
* eP: Extending the timeline, would it be feasible that the dependencies are necessary for Solid? Instead of depending on CG or whatever else...
```